src/test/java/blackjack/UserTest.java

```diff
@@ -0,0 +1,88 @@
+package blackjack;
+
+import blackjack.domain.card.Card;
+import blackjack.domain.card.CardDeck;
+import blackjack.domain.card.Symbol;
+import blackjack.domain.card.Type;
+import blackjack.domain.player.Player;
+import blackjack.domain.player.User;
+import blackjack.exception.UserNameEmptyException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class UserTest {
+    private List<Card> cards;
+    private CardDeck cardDeck;
+    private User user;
+
+    @BeforeEach
+    void setUp() {
+        cards = new ArrayList<>(
+                Arrays.asList(
+                        new Card(Symbol.HEART, Type.TEN),
+                        new Card(Symbol.SPADE, Type.QUEEN),
+                        new Card(Symbol.HEART, Type.SEVEN),
+                        new Card(Symbol.DIAMOND, Type.JACK),
+                        new Card(Symbol.CLOVER, Type.ACE),
+                        new Card(Symbol.CLOVER, Type.EIGHT)
+                )
+        );
+        cardDeck = new CardDeck(cards);
+        user = new User("pobi");
+    }
+
+    @DisplayName("user 생성 시 빈 문자열일 경우 예외 발생 확인")
+    @Test
+    void emptyStringTest() {
+        assertThatThrownBy(() -> {
+            new User("");
+        }).isInstanceOf(UserNameEmptyException.class)
+                .hasMessage("유저의 이름은 공백일 수 없습니다.");
```

Comment:
상수로 올린 에러메시지를 이용하면 에러메시지가 변경 될때마다 테스트코드를 수정하지 않아도 될 것 같아요 :) 
```suggestion
                .hasMessage(User.USER_NAME_EMPTY_EXCEPTION_MESSAGE);
```

Reply:
적용했습니다!